samples/ingestion_biru_example.py

```diff
@@ -129,4 +227,7 @@ def ingest_data(yaml_pth: Path, rsync_pth: Path) -> None:
         parser.print_help()
         sys.exit(1)
     args = parser.parse_args()
-    ingest_data(args.yaml_pth, args.rsync_pth)
+
+    script = IDSIngestionScript(args.yaml_pth, args.rsync_pth)
+    ingestible_dataclasses = script.run_extract()
```

Comment:
I think you could use the new profile-loading system here and just do this:
```python
profile = load_profile("idw")
extractor = profile.get_extractor()
ingestible_dataclasses = extractor.extract(args.yaml_path)
```

Then you wouldn't need the `run_extract()` method

Reply:
looks great!